phoenicis-repository/src/main/java/org/phoenicis/repository/dto/ApplicationDTO.java

```diff
@@ -72,6 +69,27 @@ public String getDescription() {
         return scripts;
     }
 
+    /**
+     * Returns the main miniature belonging to this {@link ApplicationDTO}.
+     * The main miniature is the miniature with the file name <code>main.png</code>.
+     * If no such miniature exists the first miniature of this {@link ApplicationDTO} is returned.
+     * If this {@link ApplicationDTO} contains no miniatures {@link Optional#empty()} is returned.
+     *
+     * @return An optional with the found URI, or {@link Optional#empty()} if no miniature exists
+     */
+    public Optional<URI> getMainMiniature() {
+        Optional<URI> result = this.miniatures.stream().filter(uri -> uri.getPath().endsWith("main.png")).findFirst();
```

Comment:
This shouldn't be serialized, which is why it has no builder method and no private field.
It's just for the benefit of accessing the main miniature.
The old way it was handled was by reordering the miniatures, so that the main miniature was in the first position, which is not intuitive.